packages/main/src/Tokenizer.js

```diff
@@ -256,9 +260,15 @@ class Tokenizer extends UI5Element {
 
 	_handleItemNavigation(event, tokens) {
```

Comment:
Open the test page, navigate to a sample containing at least two tokens. Go to a token(not the last one) via Arrow Left. Then try to navigate to a token on the right of the focused one via Arrow Down key -> does not work. The same scenario is working using Control + Arrow Down. Moreover, after an interaction with Control + Arrow Down, Arrow Down navigation works as well